src/main/webapp/script.js

```diff
@@ -216,3 +216,25 @@ function getLink(schoolName) {
       'school.main_campus,school.institutional_characteristics.level')
 }
 
+/** Adds comments to page. */
+function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comments-container');
```

Comment:
I just realized - `li` tags are only allowed to be children of `ul` or `ol` elements, not `div`. I think your easiest bet here is to create a `p` tag instead of list item.